evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -146,12 +150,18 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     onlyUnenabledAddress(_oracle)
   {
     require(oracleCount() < 42, "cannot add more than 42 oracles");
+    require(
+      oracles[_oracle].admin == address(0) || oracles[_oracle].admin == _admin,
+      "cannot modify previously-set admin address"
+    );
```

Comment:
I'm personally not a fan of moving everything into modifiers, because I find that it makes the code harder to follow (need to mentally resolve another layer of indirection, have to jump back and forth). So unless it's a complicated modifier or something that's used all over, I have a preference for just leaving the `require` explicit. 

Having said that, I'm happy to change it if you'd still like me to.

Reply:
At this point, I might be alone on this preference. We can merge without it.